src/coord/src/catalog/builtin_table_updates.rs

```diff
@@ -142,15 +144,21 @@ impl Catalog {
         oid: u32,
         schema_id: i64,
         name: &str,
+        table: &Table,
         diff: Diff,
     ) -> Vec<BuiltinTableUpdate> {
+        let persisted_name_datum = table
+            .persist
+            .as_ref()
+            .map_or(Datum::Null, |d| Datum::String(&d.stream_name));
```

Comment:
Not sure this is really any butter, but FYI you can also write this as `Datum::from(table.persist.as_ref().map(|p| p.stream_name.as_str()))`.